src/payment/payment-method.ts

```diff
@@ -7,11 +7,11 @@ export default interface PaymentMethod {
     method: string;
     supportedCards: string[];
     type: string;
+    initializationData?: any;
     clientToken?: string;
     gateway?: string;
     logoUrl?: string;
     nonce?: string;
-    initializationData?: any;
```

Comment:
To keep the file history clean, next time we could discard changes like these.

Reply:
fixed